src/state/accountCalculators.ts

```diff
@@ -103,5 +105,7 @@ export const calculateShouldRenderActionsInPositionsTable = (isCloseActionShown:
   createSelector(
     [calculateIsAccountViewOnly, calculateShouldRenderTriggersInPositionsTable],
     (isAccountViewOnly: boolean, areTriggersRendered: boolean) =>
-      !isAccountViewOnly && (areTriggersRendered || isCloseActionShown)
+      !isAccountViewOnly &&
+      ((!testFlags.isolatedMargin && (areTriggersRendered || isCloseActionShown)) ||
+        (testFlags.isolatedMargin && isCloseActionShown))
```

Comment:
[nit] i feel like if we un-inline this it will be easier to understand what cases have what results